src/main/webapp/js/instructorFeedbacks.js

```diff
@@ -184,9 +184,7 @@ function bindCopyButton() {
 }
 
 function bindCopyEvents() {
-
     $('#copyTableModal >tbody>tr').on('click', function(e) {
```

Comment:
'#copyTableModal > tbody > tr', the spaces you added on the previous pull request was better, it helps to differentiate between pseudo classes (e.g. a:link) and element > element selectors quickly just by glancing through.